README.md

```diff
@@ -16,3 +16,4 @@ The 2022 Full time grind has begun! Use this repo to share and keep track of any
 |[Fidelity](https://jobs.fidelity.com/job-details/13162494/2021-2022-undergraduates-full-stack-engineer-merrimack-nh/)| Boston + others | Full-Stack Engineer |
 |[Stytch](https://jobs.lever.co/stytch/020d5d2e-fe49-4dfd-b6c6-77a0c3eb377f) | SF, Remote | |
 |[VMWare](https://careers.vmware.com/new-graduates/jobs/R2111971?lang=en-us) | Palo Alto, Atlanta | University Launch SWE |
+|[Cruise](https://boards.greenhouse.io/cruise/jobs/2364685) | SF | |
```

Comment:
This Cruise link redirects to the main careers page for Cruise. The current new grad roles that Cruise has are at these links:
- https://boards.greenhouse.io/cruiseuniversity/jobs/3351151
- https://boards.greenhouse.io/cruiseuniversity/jobs/3351596